### PR TITLE
Add function to show compression information

### PR DESCRIPTION
**Pull request by mkindahl**
Add a function that can be used on a compressed data value to show some metadata information, such as the compression algorithm used and the presence of any null values.

### CODE REVIEW

**Comment by mkindahl** [comment]
> Useful function, let's call it somewhere in `compression_algos` so that it's covered. Should it be debug-only like the other test functions?

Yes, the tests we added were using Hyperstore, so I'll add tests for it in `compression_algos`. We don't really have a good place for utility functions and I am reluctant to place it in `public`, but maybe that is ok?

File: tsl/src/compression/compression.c [hunk 1]
```diff
@@ -1934,6 +1946,59 @@ tsl_compressed_data_out(PG_FUNCTION_ARGS)
...
+	Anum_compressed_info_algorithm = 1,
+	Anum_compressed_info_has_nulls,
+	_Anum_compressed_info_max,
+};
+
+#define Natts_compressed_info (_Anum_compressed_info_max - 1)
+
+extern Datum
+tsl_compressed_data_info(PG_FUNCTION_ARGS)
+{
```
Comment (fabriziomello) [inline tsl/src/compression/compression.c:1961]: Looks like we need some tests

**Review by fabriziomello — CHANGES_REQUESTED** [review]
We're adding some code that is not covered by any test. Is it very difficult to introduce some tests?

**Comment by mkindahl** [comment]
> We're adding some code that is not covered by any test. Is it very difficult to introduce some tests?

Nope. Based on the feedback from @akuzm I intended to add tests. Mostly wondering if there is anything else that needs to be dealt with.

**Review by fabriziomello — APPROVED** [review]

**Pull request merged**